tools/packaging/kata-deploy/helm-chart/README.md

```diff
@@ -0,0 +1,117 @@
+
+# See everything you can configure
+helm show values \
+  https://github.com/kata-containers/kata-containers/releases/download/${VERSION}/kata-deploy-${VERSION}.tgz
+```
+
+## Prerequisites
+
+- **Kubernetes ≥ v1.22** – v1.22 is the first release where the CRI v1 API
+  became the default and RuntimeClass left alpha.  The chart depends on those
+  stable interfaces; earlier clusters need feature‑gates or CRI shims that are
+  out of scope.
+
+- CRI‑compatible runtime (containerd or CRI‑O)
+
+- Nodes must allow loading kernel modules and installing Kata artifacts (the
+  chart runs privileged containers to do so)
+
+## Installing Helm
+
+If Helm is not yet on your workstation or CI runner, install Helm v3 (v3.9 or
+newer recommended):
+
+```sh
+# Quick one‑liner (Linux/macOS)
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+# Or via your package manager
+sudo apt-get install helm        # Debian/Ubuntu
+brew install helm                # Homebrew on macOS / Linuxbrew
+```
+
+Verify the installation:
+
+```sh
+helm version
+```
+
+## Installing the Chart
+
+```sh
+
+helm install kata-deploy \                           # release name
+  --namespace kube-system \                          # recommended namespa
+  https://github.com/kata-containers/kata-containers/releases/download/${VERSION}/kata-deploy-${VERSION}.tgz
+```
+
+## Updating Settings
+
+Forgot to enable an option? Re‑use the values already on the cluster and only
+mutate what you need:
+
+```sh
+# List existing releases
+helm ls -A
+
+# Upgrade in‑place, keeping everything else the same
+helm upgrade kata-deploy -n kube-system \
+  --reuse-values \
+  --set env.defaultShim=qemu-runtime-rs \
+  https://github.com/kata-containers/kata-containers/releases/download/${VERSION}/kata-deploy-${VERSION}.tgz
+```
+
+## Uninstalling
+
+```sh
+helm uninstall kata-deploy -n kube-system
+```
+
+## Configuration Reference
+
+All values can be overridden with --set key=value or a custom -f myvalues.yaml.
+
+| Key | Description | Default |
+|-----|-------------|---------|
+| imagePullPolicy | Set the DaemonSet pull policy | `Always` |
+| imagePullSecrets | Enable pulling from a private registry via pull secret | `""` |
+| image.reference | Fully qualified image reference | quay.io/kata-containers/kata-deploy |
+| image.tag | Tag of the image reference | `""` |
+| k8s.Distribution | Set the k8s distribution to use: k8s, k0s, k3s, rke2, microk8s | `k8s` |
+| env.debug | Enable debugging in the configuration.toml | `false` |
+| env.shims | List of shims to deploy | `clh cloud-hypervisor dragonball fc qemu qemu-coco-dev qemu-runtime-rs qemu-se-runtime-rs qemu-sev qemu-snp qemu-tdx stratovirt qemu-nvidia-gpu qemu-nvidia-gpu-snp qemu-nvidia-gpu-tdx` |
+| env.defaultShim | The default shim to use if none specified | `qemu` |
+| env.createRuntimeClasses | Create the k8s runtimeClasses | `true` |
```

Comment:
I guess this will only be true for the next release onwards

Reply:
Yes. The doc is in sync with the current code where we have the runtimeClasses set to true. So there is a gap I can add a comment